author.bib

```diff
@@ -815,19 +886,22 @@ @string{ Rus_D
 @string{ Ryan_C               = "Ryan, Conor" }
 
 @string{ Saeedi_S             = "Saeedi, Sajad" }
+@string{ Sagoff_A             = "Sagoff, Amelia" }
 @string{ Saha_A               = "Saha, Arindam" }
-@string{ Sahin_E              = "\c{S}ahin, Erol" }
+@string{ Sahin_E              = "{\c S}ahin, Erol" }
```

Comment:
The instructions at the beginning of this file state:

@comment{There is a known issue, if the first name starts with a non-ASCII character, the compilation for certain bibliography styles can break}

This PR escapes the first character in both first name and last name. Could you update it to:

@comment{There is a known issue, if the first or last name starts with a non-ASCII character, the compilation for certain bibliography styles can break}

Reply:
I updated the header accordingly